qt_configure.bzl

```diff
@@ -6,11 +26,29 @@ def qt_autoconf_impl(repository_ctx):
        repository_ctx: repository context
     """
     os_name = repository_ctx.os.name.lower()
-    qt_path = "dummy"
-    if os_name.find("windows") != -1:
-        qt_path = "C:\\\\Qt\\\\5.9.9\\\\msvc2017_64\\\\"
+    is_linux_machine = os_name.find("windows") == -1
```

Comment:
I think it would be better to explicitly check for both linux and windows. Something like:

```.py
if os_name.find("windows") != -1:
    ... # use default windows path(s)
elif os_name.find("linux") != -1:
    ... # use default linux path(s)
else:
    fail("Unsupported OS: %s" % os_name)
```